devices/xiaomi.js

```diff
@@ -1292,7 +1292,7 @@ module.exports = [
         model: 'LLKZMK11LM',
         vendor: 'Xiaomi',
         description: 'Aqara wireless relay controller',
-        fromZigbee: [fz.xiaomi_switch_basic, fz.xiaomi_power, fz.ignore_multistate_report, fz.on_off, fz.aqara_opple],
+        fromZigbee: [fz.xiaomi_switch_basic, fz.xiaomi_power, fz.ignore_multistate_report, fz.on_off, fz.xiaomi_basic_raw],
```

Comment:
Nice cleanup, only 1 comment here; why not keep both aqara_opple and xiaomi_basic_raw here? In this way we can support both devices with old and new firmwares.

Reply:
If I'm not wrong, you added the aqara_opple to see if it fixed the problem, that was not. I think this never used it.

Reply:
I forgot to say that I have two devices with old firmware that seems to work with the changes. I don't know if there is some other intermediate firmware.

Reply:
Yes you are right, thanks!